src/collectors/filestat/filestat.py

```diff
@@ -106,10 +160,24 @@ def get_default_config(self):
             'uid_max': 65536,
             'type_include': None,
             'type_exclude': None,
-            'collect_user_data': False
+            'collect_user_data': False,
+            'use_sudo': False,
+            'sudo_cmd': '/usr/bin/sudo',
         })
         return config
 
+    def get_output(self, fmt):
+
+        command = ['lsof', '-F', fmt]
```

Comment:
Can you use the full path for lsof?

Reply:
This PR is old :) I'm no longer using diamond..

Will it be portable to hardcode `/usr/bin/lsof` ?